src/content/docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: "NerdGraph tutorial: Topology for Applied Intelligence incident correlation"
+metaDescription: Use New Relic NerdGraph to set up or view Applied Intelligence topology, used for incident correlation.
+---
+
+This tutorial will explain how to use our NerdGraph API to set up your topology. Topology configuration allows your [Applied Intelligence decisions](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions) to better correlate incidents. 
```

Comment:
There's a great topology definition in the "Change Applied Intelligence correlation logic with decisions" doc: 

"For New Relic's Applied Intelligence, topology is a representation of your service map: how the services and resources in your infrastructure relate to one another."

I'd think it would be great to incorporate something like this definition here.